layers/meta-balena-raspberrypi/recipes-kernel/picontrol/picontrol_2020.06.16.bb

```diff
@@ -10,7 +10,8 @@ SRC_URI = " \
 	file://0002-Search-config-file-in-mnt-boot.patch \
 "
 
-SRCREV ="raspberrypi-kernel_1%9.20220728-5.10.120+revpi1"
+# hash for tag raspberrypi-kernel_1%9.20220728-5.10.120+revpi1
+SRCREV ="e2bcb3791fdc037a3631bcff50ab48fa0e474608"
 
```

Comment:
This picontrol recipe is for kernel 4x.
The picontrol recipe for kernel 5.x is https://github.com/balena-os/balena-raspberrypi/blob/master/layers/meta-balena-raspberrypi/recipes-kernel/picontrol/picontrol_2022.11.18.bb

Why did you need to modify this recipe? Makes me think that your new machine is not using the newer picontrol recipe. Please review your machine config to check